guides/common/attributes-base.adoc

```diff
@@ -77,8 +77,8 @@
 :oVirtEngine: {oVirt} Engine
 :oVirtShort: {oVirt}
 :package-clean: dnf clean
-:package-install-project: dnf install
-:package-install: dnf install
+:project-package-install: dnf install
+:client-package-install: dnf install
```

Comment:
While I think this is a better name than we had (since it's qualified at least a bit), I do wonder about what the client is. I worry that we won't have "a" client, but will always need to list a few ways for the various clients.

Reply:
Client is supposed to be a host.
And I agree that for upstream we usually need to have a list of more than a single OS, so I'm not sure how this will help.

Reply:
Can we set the value as `{package-manager} install` and have an applicable package manager attribute in `attribute-{build}`?